### PR TITLE
feat(web): add guide link for eligibility

### PR DESCRIPTION
**Pull request by antonymarion**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Feat#MANAGER-15875
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
~~- [ ] Breaking change is mentioned in relevant commits~~

### CODE REVIEW

**Review by tristanwagner — APPROVED** [review]

File: packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.controller.js [hunk 1]
```diff
@@ -17,6 +18,8 @@ export default class DomainContactDashboardCtrl {
     this.INFO_PROPERTIES = INFO_PROPERTIES;
     this.LEGAL_FORM_INDIVIDUAL = LEGAL_FORM_INDIVIDUAL;
     this.infoProperties = INFO_PROPERTIES_ARRAY;
+    this.ovhSubsidiary = this.user.ovhSubsidiary;
+    this.GUIDE_URL = GUIDE_URLS[this.ovhSubsidiary];
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.controller.js:22]:
```suggestion
    this.GUIDE_URL = GUIDE_URLS[this.ovhSubsidiary] || GUIDE_URLS.DEFAULT;
```

Comment (antonymarion) [reply packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.controller.js:22]: thks

File: packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.html [hunk 2]
```diff
@@ -8,11 +8,23 @@ <h1 data-translate="domain_tab_CONTACT_title"></h1>
...
+            </div>
+            <div class="col-md-2">
+                <oui-guide-menu
+                    data-text="{{ ::'domain_tab_CONTACT_guide_text' | translate }}"
+                >
+                    <oui-guide-menu-item
+                        data-href="{{ ::$ctrl.GUIDE_URL }}"
+                        data-external="true"
+                    ><span
+                        data-ng-bind="::'domain_tab_CONTACT_guide_url' | translate"
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/contact/dashboard/dashboard.html:23]:
```suggestion
                        data-translate="domain_tab_CONTACT_guide_manage_contact_and_services"
```

File: packages/manager/apps/web/client/app/domain/contact/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -12,5 +12,7 @@
   "domain_tab_CONTACT_description_technical": "Il gère les aspects techniques d'un service. Il est responsable de la définition des serveurs DNS autoritaires.",
   "domain_tab_CONTACT_description_billing": "Il gère uniquement la facturation d'un service dont ses prélèvements.",
   "domain_tab_CONTACT_description_error_message": "Une erreur est apparue",
+  "domain_tab_CONTACT_guide_text": "Guide",
+  "domain_tab_CONTACT_guide_url": "Gérer les contacts de ses services",
```
Comment (ghyenne) [inline packages/manager/apps/web/client/app/domain/contact/translations/Messages_fr_FR.json:16]:
```suggestion
  "domain_tab_CONTACT_guide_manage_contact_and_services": "Gérer les contacts de ses services",
```

**Review by tristanwagner — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Pull request merged**